src/constants/index.ts

```diff
@@ -1,7 +1,5 @@
 export const Q_PATH = "/qHyperCubeDef";
 
-export const DEFAULT_PAGE_SIZE = 50;
```

Comment:
There is no default page size any more. It's all resolved from the height and width of the chart rect.